nemo/collections/asr/metrics/rnnt_wer.py

```diff
@@ -106,7 +106,9 @@ def __init__(self, decoding_cfg, decoder, joint, blank_id: int):
                 decoder_model=decoder,
                 joint_model=joint,
                 blank_index=self.blank_id,
-                max_symbols_per_step=self.cfg.greedy.get('max_symbols', None),
+                max_symbols_per_step=(
+                    self.cfg.greedy.get('max_symbols', None) or self.cfg.greedy.get('max_symbols_per_step', None)
```

Comment:
+What is the difference between max_symbols and max_symbols_per_step?

Reply:
No difference. dataclass requires `max_symbors_per_step` and the config requires `max_symbols`.